infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -2,19 +2,19 @@
 x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.38.1"
+  dpl-cms-release: "2024.38.2"
 x-webmaster: &webmaster-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.35.1"
-  moduletest-dpl-cms-release: "2024.38.1"
+  dpl-cms-release: "2024.35.2"
+  moduletest-dpl-cms-release: "2024.38.2"
 # Some sites on the webmaster plan wish to track the same release as
 # sites do per default in dpl-cms-release.
 x-webmaster-with-defaults: &webmaster-with-defaults-release-image-source
   # Reference default before webmaster anchor as YAML references will not
   # override existing keys.
   <<: *default-release-image-source
-  moduletest-dpl-cms-release: "2024.38.1"
+  moduletest-dpl-cms-release: "2024.38.2"
```

Comment:
Please also update `customizable-canary` `moduletest-dpl-cms-release` to `2024.38.1`.

I do not think we should have references to older releases after this.

Reply:
I assume you mean `38.2`
That has been done now